HW4/tests/test12-deep-recursive-pack.events

```diff
@@ -0,0 +1 @@
+Snail Pack 3 Slime Pack 2 Slime Pack 3 Slime Snail Slime Pack 4 Slime Balrog Balrog Balrog
```

Comment:
Do we have a test where main pack does not end in a pack? while also having a complex pack structure like this one